src/js/controller/controller.js

```diff
@@ -255,6 +255,8 @@ define([
                 if (_model.get('state') === states.ERROR) {
                     _model.set('state', states.IDLE);
                 }
+                _model.set('preInstreamState', 'instream-idle');
```

Comment:
Yes, we need this.
This is the commit which added this line: https://github.com/jwplayer/jwplayer/commit/0ccf9c8a9bebe0371ccf9a3341c3290256b8e877